cargo-flash/Cargo.toml

```diff
@@ -24,9 +24,7 @@ sentry = ["probe-rs-cli-util/sentry"]
 env_logger = "0.10"
 colored = "2"
 probe-rs = { workspace = true }
-probe-rs-cli-util = { workspace = true, default-features = false, features = [
-    "anyhow",
-] }
+probe-rs-cli-util = { workspace = true, features = ["anyhow"] }
```

Comment:
What's the reason for changing this?

Reply:
It generated a clippy warning, because probe-rs-cli-util doesn't have any default-features.